README.md

```diff
@@ -100,10 +100,139 @@ This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0
+complicated set of packages or other dependencies.  The idea here is that
+the only thing you *really* need in your 'working' environment are
+the ISIS environment variables and the path to the ISIS executables.
+
+If the above paragraph sounded like gibberish, please seek help
+from your system administrator or local computer guru.
+
+And we can do this via customizations in the conda environment's
+activate.d/ and deactivate.d/ directories.  Adding these things can
+also be done manually from the command line, but encoding them in
+the activate.d/ and deactivate.d/ scripts is handy.
+
+1.  Create your conda environment however you like, adding whatever
+packages you need.  If you were reading the directions above, you've
+already done this.
+
+2.  Locate the path to your conda environments:
+
+        conda activate
+        echo $CONDA_PREFIX
+        conda deactivate
+
+    You'll probably get a directory that is in your home directory and
+    is named `anaconda3` or `miniconda3` or something similar.  For the
+    rest of this set of instructions, we'll refer to it as `$HOME/anaconda3`
+    to represent a directory named `anaconda3` in your home directory, but
+    this should be whatever you get from the above echo command.
+
+3.  Locate the path to your ISIS conda environment:
+
+        conda activate isis
+        echo $CONDA_PREFIX
+        conda deactivate
+
+    This should probably be `$HOME/anaconda3/envs/isis`.  You can see
+    that it starts with whatever you got from step 1, and ends in the
+    name of your isis environment, if you followed the installation
+    instructions above, you called that environment 'isis'.
+    
+    You can do the same thing to find the path to your new 'working'
+    environment, but in this example, it will be at
+    `$HOME/anaconda3/envs/working`.
+
+4.  Copy the ISIS activation and deactivation scripts to your new
+environment.  Please note that the directory names in the instructions
+below are based on how you installed conda and what you named the
+'isis' environment and the 'working' environment.  You may *not* be
+able to just copy and paste these instructions directly, they are
+an example.  Likewise, if your shell doesn't take the bash syntax
+in the .sh files, then you may need to select one of the other
+`env_vars.*` files in the isis directories.
+
+        cd $HOME/anaconda3/envs/
+        mkdir -p working/etc/conda/activate.d/
+        mkdir -p working/etc/conda/deactivate.d/
+        cp isis/etc/conda/activate.d/env_vars.sh working/etc/conda/activate.d/env_vars.sh
+        cp isis/etc/conda/deactivate.d/env_vars.sh working/etc/conda/deactivate.d/env_vars.sh
+
+5.  Edit the copied activation file in
+`$HOME/anaconda3/envs/working/etc/conda/activate.d/` to add the
+ISIS executable directory to the path, by adding this line at the
+end:
+
+        export PATH=$PATH:$ISISROOT/bin
+
+    Or whatever is appropriate for your shell if you aren't using the .sh file.
```

Comment:
I would emphasize that it's VERY important you add `$ISISROOT/bin` to the end of your path so that executables in your current environment hide executables in the ISIS environment.

Reply:
Added to new commit.